TouchPortal-API/TouchPortalAPI/__init__.py

```diff
@@ -20,40 +21,84 @@ class TYPES:
 class Client(EventEmitter):
     TPHOST = '127.0.0.1'
     TPPORT = 12136
+    RCV_BUFFER_SZ = 4096   # [B] incoming data buffer size
+    SND_BUFFER_SZ = 32**4  # [B] maximum size of send data buffer (1MB)
+    SLEEP_PERIOD = 0.01    # [s] event loop sleep between socket read events
```

Comment:
My other processes (threads) have more time to run if `SLEEP_PERIOD` is a bit larger, like `0.05`.  And I don't notice any delays with the Client I/O.  It may make sense to make this parameter tuneable per instance (eg. optional attribute to init())